### PR TITLE
Added WP on 16.04

### PR DESCRIPTION
**Pull request by ryanpq**

### CODE REVIEW

**Comment by andrewsomething** [comment]
On two out of three tries testing this, I've ended up with "Error establishing a database connection"  and the DB password was not successfully placed in the wp-config file. Looking in the cloud-init logs, I see:

```
sed: -e expression #1, char 70: unknown option to `s'
```

File: Ubuntu-16.04/web-servers/lamp-phpmyadmin.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+#cloud-config
+apt_update: true
+package_upgrade: true
+packages:
+  - apache2
+  - apache2-utils
+  - php5-mysql
```
Comment (andrewsomething) [inline Ubuntu-16.04/web-servers/lamp-phpmyadmin.yml]: @ryanpq Do we want to use the default PHP 7 here as well or is there an issue with PHPMyAdmin?

Comment (ryanpq) [inline Ubuntu-16.04/web-servers/lamp-phpmyadmin.yml]: That was not intended to be in this PR.  Still needs work and testing.  Removed.

**Comment by andrewsomething** [comment]
:+1: Once the links are update :shipit:

**Pull request merged**